analyticsdataserver/settings/local.py

```diff
@@ -89,7 +89,7 @@
 })
 
 CORS_ORIGIN_WHITELIST = (
-    'localhost:1991'
+    'http://localhost:1991',
```

Comment:
**Note to Reviewers:** Latest changes in `django-cors-headers` requires scheme in origins. Please [see](https://github.com/ottoyiu/django-cors-headers/pull/397)